mxcube3/routes/ra.py

```diff
@@ -89,27 +82,47 @@ def give_control():
 
         return make_response("", 200)
 
+    @bp.route("/update_user_nickname", methods=["POST"])
+    @server.restrict
+    def update_user_nickname():
+        """
+        """
+        name = request.get_json().get("name")
+        current_user.nickname = name
+        app.usermanager.update_user(current_user)
+        app.usermanager.emit_observers_changed()
+
+        return make_response("", 200)
+
+    @bp.route("/logout_user", methods=["POST"])
+    @server.restrict
+    def logout_user():
+        """
+        """
+        username = request.get_json().get("username")
+        user = app.usermanager.get_user(username)
+        if not user.in_control:
```

Comment:
I dont understand this `if`. If I am the one in control and I log out, I would like to kick out all the observers. But if I am _not_ in control, I would like to leave the others logged in.

This is what we do here (more or less), but of course we have users login instead of proposal.

Can the same proposal login be used at the same time?

Reply:
- username has replaced userID in the question above

-  The method above is used to force logout a user, so the if statement is just a security check to make sure that if against all odds something goes wrong no-one can force logout a user that is already in control